src/main/java/nl/tudelft/ti2206/group9/gui/GameScene.java

```diff
@@ -1,6 +1,9 @@
 package nl.tudelft.ti2206.group9.gui;
 
-import static nl.tudelft.ti2206.group9.ShaftEscape.OBSERVABLE;
+/**
+ * @author Robin, Maarten
+ */
+import javafx.event.EventHandler;
```

Comment:
Unused import. Also, the Javadoc containing `@author` was already present at lines 30-33.

<layout,documentation